R/progenySuppFunc.r

```diff
@@ -0,0 +1,205 @@
+#'This function generate a series of scatter plot with marginal distribution
```

Comment:
Here you should add a title to this function. Otherwise the whole description is taken as title